src/tsd.c

```diff
@@ -117,6 +117,9 @@ tsd_force_recompute(tsdn_t *tsdn) {
 		    <= tsd_state_nominal_max);
 		tsd_atomic_store(&remote_tsd->state, tsd_state_nominal_recompute,
 		    ATOMIC_RELAXED);
+		atomic_store_u64(
+		    tsd_thread_allocated_next_event_fastp_get_unsafe(remote_tsd),
+		    0, ATOMIC_RELAXED);
```

Comment:
Is this doing the same thing as `thread_allocated_next_event_fast_set_non_nominal(remote_tsd)`?

Reply:
Yep will fix.